src/components/Message/Message.css

```diff
@@ -0,0 +1,6 @@
+.message {
+  margin-top: 30px;
+  margin-bottom: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

Comment:
This file should be a .scss to match the others and should be imported into public/index.scss (have a look at that file to see what I mean by import).
We also probably don't want to change the font here unless it is important that it is different to fonts across the whole site. @Ruz94 What do you think?